digiroad2-oracle/src/main/resources/db/migration/V0_23__drop_node_id_constraint.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE NODE_POINT DROP CONSTRAINT NODE_POINT_FK2;
```

Comment:
Why this constraint needs to be dropped?

Reply:
NODE_ID could not exist in node point templates